src/core/qgsvectorlayertemporalproperties.h

```diff
@@ -0,0 +1,125 @@
+/***************************************************************************
+                         qgsvectorlayertemporalproperties.h
+                         ---------------
+    begin                : February 2020
+    copyright            : (C) 2020 by Samweli Mwakisambwe
+    email                : samweli at kartoza dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSVECTORLAYERTEMPORALPROPERTIES_H
+#define QGSVECTORLAYERTEMPORALPROPERTIES_H
+
+#include "qgis_core.h"
+#include "qgis_sip.h"
+#include "qgsrange.h"
+#include "qgsmaplayertemporalproperties.h"
+
+/**
+ * \class QgsVectorLayerTemporalProperties
+ * \ingroup core
+ * Implementation of map layer temporal properties for vector layers.
+ *
+ * \since QGIS 3.14
+ */
+
+class CORE_EXPORT QgsVectorLayerTemporalProperties : public QgsMapLayerTemporalProperties
```

Comment:
Remove this class please -- it's out of scope

Reply:
Yap, I thought I deleted these!!